aws/sentry/rds.tf

```diff
@@ -0,0 +1,130 @@
+  type        = string
+  description = "Name of the reader endpoint CNAME record to create in the parent DNS zone specified by `zone_name`. If left empty, the name will be auto-asigned using the format `replicas.var.postgres_name`"
+  default     = ""
+}
+
+variable "postgres_rds_instance_allocated_storage" {
+  type        = number
+  default     = 10
+  description = "The allocated storage in GBs"
+}
+
+variable "postgres_rds_instance_kms_key_id" {
+  type        = string
+  default     = ""
+  description = "KMS key ID used to encrypt SSM `SecureString` parameters"
+}
+
+variable "postgres_rds_instance_hostname" {
+  type        = string
+  default     = "db"
+  description = "The DB host name created in Route53"
+}
+
+locals {
+  postgres_rds_instance_admin_password = length(var.postgres_rds_instance_password) > 0 ? var.postgres_rds_instance_password : join("", random_string.postgres_rds_instance_password.*.result)
+  kms_key_id                           = length(var.postgres_rds_instance_kms_key_id) > 0 ? var.postgres_rds_instance_kms_key_id : format("alias/%s-%s-chamber", var.namespace, var.stage)
+}
+
+resource "random_string" "postgres_rds_instance_password" {
+  count            = var.postgres_rds_instance_enabled && length(var.postgres_rds_instance_password) == 0 ? 1 : 0
+  length           = 26
+  special          = true
+  override_special = "!#%^&*()_<>?{}[]~=+"
+}
+
+module "rds_instance_postgres" {
+  source               = "git::https://github.com/cloudposse/terraform-aws-rds.git?ref=tags/0.19.0"
+  enabled              = var.postgres_rds_instance_enabled
+  namespace            = var.namespace
+  stage                = var.stage
+  name                 = var.postgres_name
+  engine               = "postgres"
+  instance_class       = var.postgres_rds_instance_instance_class
+  database_user        = var.postgres_rds_instance_user
+  database_password    = local.postgres_rds_instance_admin_password
+  database_name        = var.postgres_db_name
+  database_port        = "5432"
+  dns_zone_id          = local.zone_id
+  vpc_id               = module.kops_metadata.vpc_id
+  subnet_ids           = module.kops_metadata.private_subnet_ids
+  security_group_ids   = [module.kops_metadata.nodes_security_group_id]
+  multi_az             = true
+  storage_type         = "gp2"
+  allocated_storage    = var.postgres_rds_instance_allocated_storage
+  storage_encrypted    = "true"
+  engine_version       = "9.6"
+  major_engine_version = "9.6"
+  db_parameter_group   = "postgres9.6"
+  publicly_accessible  = false
+  host_name            = var.postgres_rds_instance_hostname
+}
+
+data "aws_kms_key" "chamber_kms_key" {
+  count  = var.postgres_rds_instance_enabled ? 1 : 0
+  key_id = local.kms_key_id
+}
+
+resource "aws_ssm_parameter" "postgres_rds_instance_master_password" {
+  count       = var.postgres_rds_instance_enabled ? 1 : 0
+  name        = format(local.chamber_parameter_format, local.chamber_service, "sentry_postgres_password")
```

Comment:
will be have the same parameter name overridden from here https://github.com/cloudposse/terraform-root-modules/blob/sentry-rds-instance/aws/sentry/aurora-postgres.tf#L147 ?